bgpd/bgp_vty.c

```diff
@@ -19777,6 +19815,11 @@ int bgp_config_write(struct vty *vty)
 		if (CHECK_FLAG(bgp->flags, BGP_FLAG_SHUTDOWN))
 			vty_out(vty, " bgp shutdown\n");
 
+		/* Automatic RA enabling by BGP */
+		if (!CHECK_FLAG(bm->flags, BM_FLAG_IPV6_NO_AUTO_RA))
+			if (CHECK_FLAG(bgp->flags, BGP_FLAG_IPV6_NO_AUTO_RA))
```

Comment:
In what case this could be different than a global one?

Reply:
They differ if the global option is not set but the local one is set.

If the global option is defined, it propagates to all bgp routers, but nothing is printed for them. If the global option is not defined, local options are printed.

The same logic is implemented for the `bgp graceful-restart preserve-fw-state` option there.